src/app/[locale]/dataset-list.cy.tsx

```diff
@@ -0,0 +1,36 @@
+import DatasetList from './dataset-list';
+
+describe('<DatasetList />', () => {
+  context('List rendering', () => {
+    it('Shows a dataset card if one dataset is returned in the search result', () => {
```

Comment:
Or: `Shows 1 dataset card`? (the word `a` could suggest that 'at least one' card also suffices)